index.js

```diff
@@ -220,23 +220,38 @@ function fail(msg) {
 }
 
 function showHistory(friendName) {
+  if (friendName == "--chronological") {
+    return showChronologicalHistory();
```

Comment:
Should we ban friends that start with `--` ? (This feels like a hack :frowning_face: )

Reply:
+1, it's probably better to do some proper argument-parsing here.